docker-compose.yml

```
@@ -89,6 +89,17 @@ services:
     - "127.0.0.1:6391:6379"
 
 # Dependencies
+  cosmosdb-emulator:
+    image: mcr.microsoft.com/cosmosdb/linux/azure-cosmos-emulator:latest
```

The docs all specify the `:vnext-preview` tag rather than `latest` 🤔 I wonder if that's the source of the license issue you're seeing? 

```suggestion
    image: mcr.microsoft.com/cosmosdb/linux/azure-cosmos-emulator:vnext-preview
```

---

I'm very confused, I'll have to re-read it, the instructions in the larger doc that I followed are entirely different that the other one